client/src/components/RequestsTable/row.tsx

```diff
@@ -9,14 +9,38 @@ import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
 import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
 import Typography from '@mui/material/Typography';
 import { Button } from '@mui/material';
-import BadgeIcon from '@mui/icons-material/Badge';
+import httpInstance from '../../services/axiosConfig';
 import './style.css';
-import { CarWithCustomerInfo } from '../../interfaces';
+import { RowProps } from '../../interfaces';
 
-function Row(props:{ car:CarWithCustomerInfo, state:string }) {
-  const { car, state } = props;
+function Row(props:RowProps) {
+  const {
+    car, state, setCarsData, setSnackBarProperties,
+  } = props;
   const [open, setOpen] = useState(false);
 
+  const deleteCar = async (id:number) => {
+    try {
+      setSnackBarProperties((preState) => ({ ...preState, open: false }));
+      const response = await httpInstance.delete(`/cars/${id}`);
+      setCarsData((prevState) => prevState.filter(((element) => element.id !== car.id)));
```

Comment:
why put car.id       setCarsData((prevState) => prevState.filter(((element) => element.id !== car.id)));
you already have car id as parameter here  const deleteCar = async (id:number) => {
 setCarsData((prevState) => prevState.filter(((element) => element.id !==  id)));